src/client/pages/CubeDraftPage.tsx

```diff
@@ -10,34 +10,23 @@ import Pack from 'components/Pack';
 import RenderToRoot from 'components/RenderToRoot';
 import { CSRFContext } from 'contexts/CSRFContext';
 import { DisplayContextProvider } from 'contexts/DisplayContext';
-import CardType from 'datatypes/Card';
 import Cube from 'datatypes/Cube';
 import Draft, { DraftStep } from 'datatypes/Draft';
 import DraftLocation, { addCard, location, removeCard } from 'drafting/DraftLocation';
 import { locations } from 'drafting/DraftLocation';
 import useLocalStorage from 'hooks/useLocalStorage';
 import CubeLayout from 'layouts/CubeLayout';
 import MainLayout from 'layouts/MainLayout';
-import { cardCmc, cardType, makeSubtitle } from 'utils/cardutil';
+import { makeSubtitle } from 'utils/cardutil';
 
-import { setupPicks } from '../../util/draftutil';
+import { getCardDefaultRowColumn, setupPicks } from '../../util/draftutil';
 
 interface CubeDraftPageProps {
   cube: Cube;
   draft: Draft;
   loginCallback?: string;
 }
 
-const getCardsDeckStackPosition = (card: CardType): { row: number; col: number } => {
```

Comment:
Moved into draft utils and updated logic to use cmcColumn from getDefaultPosition (which is used in grid draft)